samples/nrf9160/serial_lte_modem/src/slm_at_host.c

```diff
@@ -280,6 +280,7 @@ static void cmd_send(struct k_work *work)
 	if (err) {
 		LOG_WRN("UART RX failed: %d", err);
 	}
+	k_sleep(100);
```

Comment:
Why did we add this `k_sleep` here and on line 365?

Reply:
uart_rx_enable() and uart_rx_disable() are asynchronous API. Maybe not needed for rx_enable() but definitely need for rx_disable(), allowing time for Nordic code in Serial to complete disabling properly. There is a discussion with low level team in previous PR1933 on this. As we still see occasional RX problem, I added the two sleep just in case.

Reply:
@lemrey this change is removed as it does not fix the issue. We'll handle the UART problem in a JIRA ticket or else way.